planemo/runnable.py

```diff
@@ -212,31 +194,26 @@ def workflow_dir_runnables(path: str, return_all: bool = False) -> Optional[Unio
     return None
 
 
-def tool_dir_runnables(path: str, temp_path: Optional[str]) -> Optional[List[Runnable]]:
+def tool_dir_runnables(path: str) -> List[Runnable]:
     tool_sources = [p for p in yield_tool_sources_on_paths(ctx=None, paths=[path])]
```

Comment:
Might as well just do a single list comprehension ?
```suggestion
    return [for_path(p) for (p, _) in yield_tool_sources_on_paths(ctx=None, paths=[path])]
```